static/js/components/CreatePostForm.js

```diff
@@ -96,6 +105,17 @@ export default class CreatePostForm extends React.Component {
                   onChange={onUpdate}
                 />
               </div>}
+            <div className="row channel-select">
+              <label>Channel</label>
+              <select
+                onChange={updateChannelSelection}
+                name="channel"
+                value={channel ? channel.name : ""}
+              >
+                <option label="Select a channel" value="" />
```

Comment:
If I select this option it causes a page refresh. Can you look into it?

Reply:
oh yeah I know what's up